Documentation warning

## Issue Details
In the overnight build of the master version we see the waring:

> /home/cgal-testsuite/cgal_doc_build/CGAL-4.14-I-47/include/CGAL/Three/Triangle_container.h:94: warning: Member getShrinkFactor() (function) of class CGAL::Three::Triangle_container is not documented.

The reason for this is not 100% known, looks like a small quirk in doxygen for which an easy workaround exists.

## Source Code

Current code in `Three/Triangle_container.h`:
```
  ///@{
  //! getter for the "shrink_factor" parameter
  float getShrinkFactor();
  //! getter for the "plane" parameter
  QVector4D getPlane();
```

Replacement:
```
  ///@{

  //! getter for the "shrink_factor" parameter
  float getShrinkFactor();
  //! getter for the "plane" parameter
  QVector4D getPlane();
```

So just an empty line after the `@{`

--- a/Three/include/CGAL/Three/Scene_item_rendering_helper.h
+++ b/Three/include/CGAL/Three/Scene_item_rendering_helper.h
@@ -78,12 +78,6 @@ public:
 #endif
   Q_DECLARE_FLAGS(Gl_data_names, Gl_data_name)
 
-#ifdef DOXYGEN_RUNNING
-  //! \brief Flag interface for Scene_item::Gl_data_name.
-  //! \todo Review Laurent Rineau:  Should be explained better. Points to `QFlags`...
-  enum Gl_data_names{};
-#endif
-  
   QMenu* contextMenu() Q_DECL_OVERRIDE;
 
   /*!

--- a/Three/include/CGAL/Three/Triangle_container.h
+++ b/Three/include/CGAL/Three/Triangle_container.h
@@ -90,6 +90,7 @@ struct DEMO_FRAMEWORK_EXPORT Triangle_container :public Primitive_container
   /// If the shaders of this program doesn't need one, you can ignore it.
   /// The others should be filled at each `draw()` from the item.
   ///@{
+
   //! getter for the "shrink_factor" parameter
   float getShrinkFactor();
   //! getter for the "plane" parameter

